icons/discordjs/discordjs-original-wordmark.svg

```diff
@@ -0,0 +1,9 @@
/IDwBDhF6KogpFAqFQqG4Es1n/XZ07NZg6BE4b8XMMpGILT6XWGKSYTiXjLyrtyzZBYJFb+Q0aqVOGRyg1tVpxim9HuUU2t5FJcqcahMLwEsCvEXAi8ivOgJ4S4DaxsfPWVn3yTzk2evrlxdV9wbhPh/3WbtNwap6VsZOCbIwNrCAkZMFs/13Q5Fgvu/niS2JiEKcTTRf4K2FF4wFFy6kJW65mZS02IRSBHGHDv/y9qnfMSAgpGsrjTq4lYLTduMU2h4Mwza9u221BPhKgJ8E+EiAf5Hw9XZCsoaRL1w+Zr3fa7grQk8FMYVCoVAoFFeiBKDdNEG/TKvwHuXuzm0MkM8CeQogjwMKWLnGlj294GFhIUcoVQTQSoBOAjREFr/2Vy9Jfq+sZlPz48bO2d5xBypWao0BoI583rBMzXm5YtW2EogoFsomVt7MLGBiZP9ZWHkmGn/vl7WCkAICUgiAYxjGt2hinBKQxa4Ksh91UrHfHLdScn6dgkSk7K1Xfgz7NWbuddyVv02XbqZQKBQKheJKBADc0bTN8/o3nzyIAePlzs4VRM4rDS6lloPEyGFaKwPYSomIKkmRCJYqH0a2CIbIOds7RkFOKaiQ5OsVPFwJ4p5gJgc5KvswJR54RhbHAgCRhRpg6sibjJIUCWHi2UjsTf3Vd3+NmXsLpeRu0wgxhUKhUCgUV6MEoF09+uZbAdq6//a0Me6GgOj3Xf7jiYUx066i4pUlAEAdgADNr2OvfOqjDpzrLPtqAxabYUv4Wp8XIZe6u0cQu6S+HoVCoVAoFIoDAgD+X1F9/iuIlmOeNsbd5Jpu/HthzLTrqFhVCUf4PORZxkUEzcs2p78OOdpMeQCiJFxed3Hh65BvRkrdB1QQUygUCoVCcQfCDX2SJTp59UsSkfI8bYy7MAn6jZM3NfwDshir6CIcdgjk6hTmFzY2WpZhSJoGeXIYpQwkIuUdux45eW3sJ9m4zw0JFcQUCoVCoVDcgQjA+u2J6Vev5cVOQeWWLK4WiJJw9Y/zH7+J+0QmKwgPwDIjsvWWuNsHnxUlIdmJbdcYCCHma3mxU/7v73GJkI+3Mm9IXF7+hEKhUCgUCqUICQB2Xl6cPrDltNteSr/BqKHzmSQiZcfc3DF6yYk30lCxmsMPQgRA9ib/kXkt7+KaLvWf8VVxms5O7qPaQkBM1/LjXnh1Z+f9kP1/3xuSGnkQUigUCoVCqdKoAKhXhKfPCPFqsBA1TI8QkMKErL/D347qcxLOjw7fDYcif/4+Iim8rk/Lbxkw/i7sr8pDQIxXc89NenXn43Yx/MCazzRlgkKhUCgUirvhAVinbG74W47p5oeo/BLAVQYCYryUEzOhSAw/MDLpBETIots0bUurjYdTNz/Ni5a/XdxnlUWSxJRzt/Y9Vx4xDNSwOzIKhUKhUCjVChUA9dKRSaPrez/yMwCtpw2qDBKkWxczj0x+a89TJ1AOMeZE7kSLfxme8Fwj79ZfsCzXyM02eAyrzbxvVfxncyISvspEOf1PBTGFQqFQKBRPogKg/m7Isd6tA7svZRi2nqcNqgg2kT+/O+WPiT8em50Cz4hhOwzkus+avi0n+c7u9O2b/po6LwHQeMged2DOMd/89qWort/r9TcskH1fLv/TSXUUCoVCoVA8iQgAu68svcGxmo2tgrq2U7DK5p42qhwQk1C4eumJN6avjP/0JiogxlyACMCWkhvPb7yw4HAj3/aRdXSNA5Wc+hHUMO3HS5Yjp9K2P//aru5beV5vgXwzUu76zDRCTKFQKBQKpSrAAVD7wlfzU/j5acG6hh+BYfw8bdT9IES6mVZw4Y052x/bjeKocGVrDTsTe7SYA6B6O2xVs+4Nh77ipfSbxDBM9U5PkcTrmaa0b16P7LS6EIX2mxABFazmQQUxhUKhUCiUqgILWcCpX+n2Y6OnW0z9TKvwHomqVwTAahTylq9L/PariLjPc1BcScLZpdWcRQlhPKXjF6FDW02f7aMK/gfLsAEetq1ciETMuG1KW7Tg8Pg/E26fNKF44mKlJmZSQUyhUCgUCqWqoQSgAKD6elB05zaB3T9Scdr+njYKgM0kFK4/lhL59YKYqakoTo+oSlHh+1FCGLcP6at9PezngcGaxmM0Sq8BqLqTGokgWWJyjDf+XHB0+qaE29FmyH63wUlVPKggplAoFAqFUhUpId6+HHDg8VZBnWfrlL4j4eYJYiIRMwx8zrrYzP0rvzr0/FUUi+CqHBW+H46+VQDgRrV9y29Y61eGB2nrjVJyml5F73sUkYhpJlvhjqTM4ys+ih56EUW50XCB76kgplAoFAqFUpUpIYynd/4ypE/TSSP81SEjlZymJ1w0SUwkYprVZvw7y5S29Z3oQfv1+hsCqr8QLg0WRaLY/tq35STvsW3eeiJY16iXTuEbpmBVnSBXA3E1Fl60nDHy+XtT8hN2fbBvQCJKimAbXFSzmgpiCoVCoVAo1YF7oppTH50fFNZ8TO9ATWgvtcKrq4JVtmTA+JS3YYlI2ZJkSxKI5aqBL4i9nHc6ev6B8GTI4kuCLMZqmhAuDUdxzBX9zvZtOcl7VOs3uweqQ1trlN4tVKy6Bcsqm3MM1xAVy+8WJUi3JNGWbhUtF01CwbnrhUmxa+I/vHA+K8YKN4lgR6ggplAoFAqFUp1gijZH4Wb/G/tuz5WN6vm1bqZVeQeoWE2AklNoJEIEAJCIKNgk3sTbTAVWm8lkEU3GM1kHUyPiPs+DLLpI0SY6bPa/1zYYyGKXQ0k/33mvS5NB2qHNX27uxfl4+2j8fZSM1lulUHtxrFLHMkpfUeLziSTaBMIbBdFqNovGwvT8hPQt8T9kJBvieJT0raPfbSjeF277shQKhUKhUCjVEcZhA+QIsl3EPUzk0h79lVA8OYvAzWKsGnC3n4Hy+9pOaT4HPOx3KogpFAqFQqHUFByjmg+jcexRydoaBa4M5fW1nSrp8/8HNqVuTuHkpk0AAAAASUVORK5CYII="/>
```

Comment:
Hmm, this looks like an SVG loading an image. It'd be better if it's a good SVG. 🙂

Reply:
Fixed :D